EncoreTickets.SDK.Tests/AssertExtension.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using NUnit.Framework;
+
+namespace EncoreTickets.SDK.Tests
+{
+    internal static class AssertExtension
+    {
+        public static void PropertyValuesAreEquals<T>(T expected, T actual)
+        {
+            var properties = typeof(T).GetProperties();
+            foreach (var property in properties)
+            {
+                var expectedValue = property.GetValue(expected, null);
+                var actualValue = property.GetValue(actual, null);
+
+                if (!(actualValue is IEnumerable && !(actualValue is string)) && !Equals(expectedValue, actualValue))
```

Comment:
Doesn't that check mean that if the ```actualValue``` is an IEnumerable and not a string, then regardless of ```expectedValue``` we will not hit ```Assert.Fail```? (Effectively ```Assert.Extension.PropertyValuesAreEquals(new List<string>{ "c", "d", "e" }, new List<string>{ "a", "b" })``` would not fail)

Reply:
Yes, this method to assert that shallow properties are equal, I need to rename it.